src/main/java/hellos/Hellos.java

```diff
@@ -43,12 +41,18 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+
+  private String JadynSaysHello() {
```

Comment:
The "common standard" is that methods in Java start with a lowercase letter, so `jadynSaysHello` instead of `JadynSaysHello`.